src/version.ts

```diff
@@ -1,2 +1,3 @@
 // modified by build process.
 export const VERSION = '2.8.0'
```

Comment:
it says in Readme changes 2.7.0...2.7.1 , but VERSION says 2.8.0
is it OK, @nicktaras

Reply:
This is ok for staging - we can update when we move to release 2.8.0